### ISSUE TITLE
g310a test failure

### ISSUE THREAD

**Issue by joebo** [jsoftware/j-playground#38]
[Playground Link](https://jsoftware.github.io/j-playground/bin/html2/#url=https://raw.githubusercontent.com/jsoftware/j-playground/j903_wasm/test/g310a.ijs)
```
   NB. : -------------------------------------------------------------------
   
   jnc   =: 4!:0
   verb  =: 3&= @ jnc
   boxed =: 32&=  @ type
   open  =: 32&~: @ type
   inv   =: 1 : 'u^: _1'
   
   pow=: 2 : 0
 assert. verb <'u'
 if.     verb <'v' do. u powv v 
 elseif. boxed n   do. u powg n
 elseif. 1         do. u pown n
 end.
)
   
   powv=: 2 : 0
 u pown (v y) y 
  :
 x u pown (x v y) y 
)
   
   pown =: 2 : 0
 if.     #$n do. u powg (>@{.`(>@{:))"1 n;"0<y
 elseif. 0>n do. u f. inv pown (-n) y
 elseif. 1 do.
  assert. n-:<.n
  z=.y [ t=.0-:y [ i=._1
  while. (t-:z)<n>i=.>:i do. z=.u t=.z end.
 end.
  :
 x&u pown n y
)
   
   powg=: 2 : 0
 assert. ($n) -: ,2
 v0=. (0{n) `:6
 v1=. (1{n) `:6 
 u pown (v0 y) v1 y
  :
 assert. ($n) -:"1 ,. 2 3
 n=. _3}.'[';n
 v0=. (0{n) `:6
 v1=. (1{n) `:6
 v2=. (2{n) `:6
 (x v0 y) u pown (x v1 y) (x v2 y)
)
   
   (>: pow n -: >: ^: n) 1 [ n=:?10
1
   (>: pow n -: >: ^: n) 1 [ n=:?10$10
1
   (>: pow n -: >: ^: n) 1 [ n=:_5+?10$10
1
   (>: pow n -: >: ^: n) 1 [ n=:i.9
1
   
   ((0&,+,&0) pow (i.9) -: (0&,+,&0)^:(i.9)) ,1
1
   
   f=: (0&,+,&0) pow (i.9)
   3 -: 4!:0 <'f'
1
   
   (+/\ pow n -: +/\ ^:n) 7{.1 [  n=:i.9
1
   (+/\ pow n -: +/\ ^:n) 7{.1 [  n=:_7+i.15
1
   g=: i.@>:@+:@# - #
   (+/\ pow g -: +/\ ^:g) 7{.1 
1
   
   (2&o. pow _ -: 2&o. ^:_) 1
1
   
   f=:(#:1 3)&(+/ .*)
   (f pow n -: f ^: n) 0 1 [ n=:_5+i.11
0
|assertion failure: RUN1
|[-67] //test/g310a.ijs

```

**Comment by joebo** [comment]
As a clue -- This test script does NOT fail under the linux_wasm build so it's more likely it is something with the wasm / javascript runtime.

**Comment by joebo** [comment]
fixed with 517e00a2f9c77a2cbe65bd89d624331fb90fe563
